.gitignore

```diff
@@ -752,3 +752,6 @@ __azurite_db_blob_extent__.json
 __blobstorage__/
 docker-compose.local.override.yml
 __azurite_db_*
+
+# Pulumi
+Pulumi.dev.yaml
```

Comment:
might be easier to provide a stack file?
Describe stack in readme or pass in params when calling pulumi CLI

Reply:
done